desktop/apps/artist/components/current_venice_feature/index.coffee

```diff
@@ -0,0 +1,42 @@
+_ = require 'underscore'
+Q = require 'bluebird-q'
+request = require 'superagent'
+cache = require '../../../../lib/cache.coffee'
+
+cacheKey = "venice-2017"
+
+fetchVeniceArtists = ->
+  Q.promise (resolve, reject) ->
+    cache.get cacheKey, (err, cachedData) ->
+      return reject(err) if err
+      return resolve(JSON.parse(cachedData)) if cachedData
+
+      request
+        .get "http://files.artsy.net/force/venice_2017_artists.json"
```

Comment:
Yea, seems 👍 to me. I guess it doesn't matter that the file isn't SSL, but we should maybe look into that at some point on vanity's S3 folder (or cloudfront distro or whatever).